src/LiveQuery/ParsePubSub.js

```diff
@@ -1,26 +1,45 @@
-import { RedisPubSub } from './RedisPubSub';
-import { EventEmitterPubSub } from './EventEmitterPubSub';
+import { loadAdapter } from '../Adapters/AdapterLoader';
+import {
+  EventEmitterPubSub
+} from '../Adapters/PubSub/EventEmitterPubSub';
+
+import {
+  RedisPubSub
+} from '../Adapters/PubSub/RedisPubSub';
 
 let ParsePubSub = {};
 
+function usePubSubAdapter(config: any): any {
```

Comment:
Minor but I guess the return type should be void?
Also, can't see where this is used